examples/14_2_mono_camera_control.py

```diff
@@ -74,17 +109,54 @@ def displayFrame(name, frame):
         displayFrame("right", frameRight)
         displayFrame("left", frameLeft)
 
-   
-        
         # Update screen
         key = cv2.waitKey(1)
         if key == ord('q'):
             break
+        elif key == ord('c'):
+            ctrl = dai.CameraControl()
+            ctrl.setCaptureStill(True)
+            controlQueue.send(ctrl)
+        elif key == ord('t'):
+            print("Autofocus trigger (and disable continuous)")
+            ctrl = dai.CameraControl()
+            ctrl.setAutoFocusMode(dai.CameraControl.AutoFocusMode.AUTO)
+            ctrl.setAutoFocusTrigger()
+            controlQueue.send(ctrl)
+        elif key == ord('f'):
+            print("Autofocus enable, continuous")
+            ctrl = dai.CameraControl()
+            ctrl.setAutoFocusMode(dai.CameraControl.AutoFocusMode.CONTINUOUS_VIDEO)
+            controlQueue.send(ctrl)
+        elif key == ord('e'):
+            print("Autoexposure enable")
+            ctrl = dai.CameraControl()
+            ctrl.setAutoExposureEnable()
+            controlQueue.send(ctrl)
+        elif key in [ord(','), ord('.')]:
+            if key == ord(','): lensPos -= lensStep
```

Comment:
Afaik you can only control lens position -> focus on color camera, so this won't actually do anything. Same for autofocus control

Reply:
yes, it does nothing, fixed it